### PR TITLE
Add case study: Booking.com

### PR DESCRIPTION
**Pull request by jkld11**
Adding Booking.com case study (featured).
Moving Booz Allen to featured. 
Removing Nav and Spotify from featured.

### CODE REVIEW

**Comment by sftim** [comment]
@jkld11 you don't say, but I presume this is based on https://www.cncf.io/case-study/bookingcom/

File: content/en/case-studies/booking-com/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    <h2>Solution</h2>
+    After a year operating OpenShift, the platform team decided to build its own vanilla Kubernetes platform—and ask developers to learn some Kubernetes in order to use it. “This is not a magical platform,” says Ben Tyler, Principal Developer, B Platform Track. “We’re not claiming that you can just use it with your eyes closed. Developers need to do some learning, and we’re going to do everything we can to make sure they have access to that knowledge.”
+​
+<h2>Impact</h2>
+   Despite the learning curve, there’s been a great uptick in adoption of the new Kubernetes platform. Before containers, creating a new service could take a couple of days if the developers understood Puppet, or weeks if they didn’t. On the new platform, it can take as few as 10 minutes. About 500 new services were built on the platform in the first 8 months.
+</div>
+</div>
+</section>
+<div class="article image overlay" style="background-image: url('/images/booking2.JPG');">
```
Comment (sftim) [inline content/en/case-studies/booking-com/index.html:43]: Could this be `/images/booking2.JPG`?

File: content/en/case-studies/booz-allen/index.html [hunk 2]
```diff
@@ -4,7 +4,11 @@
 case_study_styles: true
 cid: caseStudies
 css: /css/case-studies-gradient.css
-featured: false
```
Comment (sftim) [inline content/en/case-studies/booz-allen/index.html:7]:
```suggestion
featured: true
```
to be explicit about this(?)

**Comment by jkld11** [comment]
> @jkld11 you don't say, but I presume this is based on https://www.cncf.io/case-study/bookingcom/

Yes, correct! I'll add links in future.

**Comment by alexcontini** [comment]
@sftim just talking to Julie about squashing commits. I have been asked to do that before too and I can't figure out how to, as we are both using the browser or desktop version of github, do you know if there is a way to do it using those versions? 

Sorry for asking, I get asked sometimes and it seems like I can't do it for some reason, I could be very wrong

File: content/en/case-studies/booz-allen/index.html (hunk 2 above)
Comment (jkld11) [reply content/en/case-studies/booz-allen/index.html:7]: See below, `featured: true` is in line 8.

**Comment by sftim** [comment]
@jklkd11 I think I have enough access to do the squash on your behalf. Is it OK if I try that for you?

**Comment by jkld11** [comment]
> @jklkd11 I think I have enough access to do the squash on your behalf. Is it OK if I try that for you?

Yes please, I'd appreciate it. Thanks!

**Pull request merged**